authentication/java/azure/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/Eclipse Foundation/jdk-16.0.2.7-hotspot
```

Comment:
There are a few new project files in the java/azure folder. Im not sure if this is intended?

Reply:
these files keep being created by the Java VS code extension, and from what I read you're [supposed to commit them](https://github.com/microsoft/kiota/pull/573/commits/0506d89e3ff9dd062d0d05b32f695e3fd5f4a4bb). But I can see the prefs file contain a hard path to java binaries, which is a big red flag. I'll remove and exclude the pref files, thanks for bringing this up.

Reply:
solved via 41c5ee2 and a641f63 (push incoming)